js_modules/dagit/packages/app/src/NUX/CommunityNux.tsx

```diff
@@ -20,13 +21,20 @@ export const CommunityNux = () => {
     'communityNux',
     (data) => data,
   );
-  if (didDismissCommunityNux) {
+  const {data, loading} = useQuery(GET_SHOULD_SHOW_NUX_QUERY);
+  const [dismiss] = useMutation(SET_NUX_SEEN_MUTATION);
+
+  if (!isLocalhost()) {
+    return null;
+  }
+  if (didDismissCommunityNux || loading || (data && !data.shouldShowNux)) {
     return null;
   }
   return (
     <CommunityNuxImpl
       dismiss={() => {
         dismissCommunityNux('1');
+        dismiss();
```

Comment:
names here are a bit confusing - this is dismissInBrowser and dismissOnServer maybe?

Do we still need the browser one once we have the server one and its only on for localhost? Certainly can't hurt...

Reply:
Yeah we don't really need the browser one... but yeah figured it can't hurt